pkgdb/CONTRIBUTING.md

```diff
@@ -10,7 +10,15 @@ pkgdb
 $ make check;
 ...<SNIP>...
 ```
+## Integration Tests
 
+```shell
+$ nix develop;
+$ cd ./pkgdb;
+$ make -j all;
+$ make bats-check;
+$ command -v flox-pkgdb-tests; #to run the bats tests as an executable
```

Comment:
This command just sanity checks that the test wrapper is in `PATH`.

You can run it by name

Reply:
should be resolved with https://github.com/flox/flox/pull/594/commits/b054862d380dcdcb178851dbf2f6d1017e36afdc